File: Tests/SwiftKotlinFrameworkTests/Tests/extensions.kt
var Double.km: Double {
    return this * 1000.0
}
var Double.m: Double {
    return this
}
open fun Double.toKm() : Double {
    return this * 1000.0
}
fun Double.toMeter() : Double {
    return this
}
public fun Double.Companion.toKm() : Double {
    return this * 1000.0
}
public var Double.Companion.m: Double {
    return this
}

File: Tests/SwiftKotlinFrameworkTests/Tests/extensions.swift
extension Double {
    var km: Double { return self * 1000.0 }
    var m: Double { return self }
}
extension Double {
    func toKm() -> Double { return self * 1000.0 }
    func toMeter() -> Double { return self }
}
extension Double {
    static func toKm() -> Double { return self * 1000.0 }
    static var m: Double { return self }
}
